### DISCUSSION TITLE
Issue when using chatterino in fullscreen on mac

### DISCUSSION THREAD

**Discussion by whoy51** [Chatterino/chatterino2#5567] [Q&A] [answered]
Hello.

I am a user of Chatterino and I like using the app in fullscreen right next to the stream I am watching. However, when I click on someone's user card, it opens in a completely separate fullscreen window, instead of just being a pop-up like on Windows. I wonder if there's a fix for this, or if this bug needs to be fixed.

Ex: When I click on my user card from chat, it goes into a separate window on a fullscreen view. I would just like it to be a small popup, like the behavior on Windows.
<img width="2560" alt="image" src="https://github.com/user-attachments/assets/77d0f069-6c6d-413f-bb5d-238dc6cc5b0b">

**Comment by pajlada** [comment, marked as answer]
Created an issue as a bug report at https://github.com/Chatterino/chatterino2/issues/6936 - thanks for the report
